### ISSUE TITLE
tf.image.adjust_hue runs incorrectly on TensorFlow 2.4 version

### ISSUE THREAD

**Issue by triumph-wangyuyang** [tensorflow/tensorflow#58698]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Documentation Bug

### Source

source

### Tensorflow Version

TF 2.4

### Custom Code

Yes

### OS Platform and Distribution

windows 11

### Python version

3.8.15

### CUDA/cuDNN version

CUDA: 11.3 cuDNN cuDNN 8.6.0

### Current Behaviour?

```shell
When running the tf.image.adjust_hue operator test code on TensorFlow 2.4, the official requirement that the parameter delta must be in the interval [-1, 1], but when I set it to not be in this interval, it can still run successfully.
```

### Standalone code to reproduce the issue

```shell
import tensorflow as tf

    image = [[[1, 2, 3], [4, 5, 6]],
             [[7, 8, 9], [10, 11, 12]],
             [[13, 14, 15], [16, 17, 18]]]
    image = tf.constant(image)
    print(tf.image.adjust_hue(image, 12.0))
```

### Relevant log output

```shell
result: tf.Tensor(
[[[ 1  2  3]
  [ 4  5  6]]

 [[ 7  8  9]
  [10 11 12]]

 [[13 14 15]
  [16 17 18]]], shape=(3, 2, 3), dtype=int32)
```
</details>

**Comment by tilakrayal** [comment]
@triumph-wangyuyang,
 It's unlikely for TF 2.4 version to receive any bug fixes except when we have security patches. There is a high possibility that this was fixed with later TF versions. Perhaps you can use the latest tf versions for your case.
 I tried in tf v2.10 and was able to get the expected output. Kindly find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/9daf47fea5a016bb64e5e867dc98e0b9/untitled780.ipynb). Thank you!

**Comment by triumph-wangyuyang** [comment]
> @triumph-wangyuyang, It's unlikely for TF 2.4 version to receive any bug fixes except when we have security patches. There is a high possibility that this was fixed with later TF versions. Perhaps you can use the latest tf versions for your case. I tried in tf v2.10 and was able to get the expected output. Kindly find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/9daf47fea5a016bb64e5e867dc98e0b9/untitled780.ipynb). Thank you!

Thank you for your reply, because the experimental environment requires tensorflow version 2.4, and then I found some problems in this version of the test, I will try to migrate the experiment to the higher version of tensorlfow later.

**Comment by tilakrayal** [comment]
@triumph-wangyuyang,
From the [gist](https://colab.research.google.com/gist/tilakrayal/9daf47fea5a016bb64e5e867dc98e0b9/untitled780.ipynb) it was clear that the issue was resolved in the latest tf v2.10 and please feel free to move this to closed status. Thank you!

**Comment by triumph-wangyuyang** [comment]
> @triumph-wangyuyang, From the [gist](https://colab.research.google.com/gist/tilakrayal/9daf47fea5a016bb64e5e867dc98e0b9/untitled780.ipynb) it was clear that the issue was resolved in the latest tf v2.10 and please feel free to move this to closed status. Thank you!

thank you for your reply
